recordings/conftest.py

```diff
@@ -103,7 +105,9 @@ def places(countries_in_db):
 @pytest.fixture
 def recordings(dialects_in_db, places_in_db, audio_files):
     """ Return Recording instances which haven't been saved yet. """
-    return (
-        Recording(dialect=dialects_in_db[0], place=places_in_db[0], recording=File(audio_files[0])),
-        Recording(dialect=dialects_in_db[1], place=places_in_db[1], recording=File(audio_files[1])),
-    )
+    with File(audio_files[0]) as f0:
+        with File(audio_files[1]) as f1:
+            yield (
+                Recording(dialect=dialects_in_db[0], place=places_in_db[0], recording=f0),
```

Comment:
`with` and `yield` here could be problematic https://stackoverflow.com/questions/41881731/is-it-safe-to-combine-with-and-yield-in-python

Reply:
This is OK, it is a [pytest idiom](https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code).